website/source/api/agent/connect.html.md

```diff
@@ -8,7 +8,7 @@ description: |-
 
 # Connect - Agent HTTP API
 
-The `/agent/connect` endpoints interact with [Connect](/docs/connect/index.html)
+The `v1/agent/connect` endpoints interact with [Connect](/docs/connect/index.html)
```

Comment:
Hey thanks for this although the version prefix is intentioally omitted on all out API docs in theory. We mention this in: https://www.consul.io/api/index.html#version-prefix.

For example check any other API docs page - none of the URLs have /v1/ prefix. https://www.consul.io/api/catalog.html. We could potentially update the wording on the link above to make that more obvious but I don't want to change that in this file and make it inconsistent with all other docs.

Reply:
![image](https://user-images.githubusercontent.com/120915/56807423-a652c900-6826-11e9-877f-138a2bc58501.png)

Random example.

Reply:
If you want to update this PR so the commit retains your autorship I'll merge the curl fix right away but it's also trivial so if you happy to do it if you don't have time!

Reply:
> Hey thanks for this although the version prefix is intentioally omitted on all out API docs in theory. We mention this in: https://www.consul.io/api/index.html#version-prefix.
> 
> For example check any other API docs page - none of the URLs have /v1/ prefix. https://www.consul.io/api/catalog.html. We could potentially update the wording on the link above to make that more obvious but I don't want to change that in this file and make it inconsistent with all other docs.

Thanks for pointing this one out, I shall remember in future :)